server/src/repositories/asset-job.repository.ts

```diff
@@ -163,6 +164,84 @@ export class AssetJobRepository {
       .execute();
   }
 
+  @GenerateSql({ params: [DummyValue.UUID] })
+  getForAssetDeletion(id: string) {
+    return this.db
+      .selectFrom('assets')
+      .select([
+        'assets.id',
+        'assets.isVisible',
+        'assets.libraryId',
+        'assets.ownerId',
+        'assets.livePhotoVideoId',
+        'assets.sidecarPath',
+        'assets.encodedVideoPath',
+        'assets.originalPath',
+      ])
+      .$call(withExifInner)
```

Comment:
I don't think a query for an asset delete should inner join on anything. We shouldn't _not_ delete it, because it is missing exif. That may, in fact, be the reason we're trying to delete it.

Reply:
True that makes sense, mb.